pkg/controllers/csi/provisioner/controller.go

```diff
@@ -223,8 +233,7 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(
 			// reporting error but not returning it to avoid immediate requeue and subsequently calling the API every few seconds
 			return true, nil
 		} else if updatedDigest != "" {
-			dynakubeMetadata.LatestVersion = ""
-			dynakubeMetadata.ImageDigest = updatedDigest
+			tenantConfig.DownloadedCodeModuleVersion = ""
```

Comment:
now the `ImageDigest` and `LatestVersion` is combined into the `DownloadedCodeModuleVersion`, so setting it here to empty is not what we want

this logic is no longer relevant, it used to be that in case `ImageDigest` was set we did things differently (because unpacked images where stored in a different place, but now they live next to the unpacked zips)

TLDR: you should always set the `DownloadedCodeModuleVersion` to either the `updatedDigest` or `updateVersion`

Reply:
Fixed (I think?)